test-app/tests/integration/components/toucan-form/toucan-form-test.gts

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-undef -- Until https://github.com/ember-cli/eslint-plugin-ember/issues/1747 is resolved... */
+/* eslint-disable simple-import-sort/imports,padding-line-between-statements,decorator-position/decorator-position -- Can't fix these manually, without --fix working in .gts */
+import { click, fillIn, render } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+
+import ToucanForm from '@crowdstrike/ember-toucan-form/components/toucan-form';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+
+module('Integration | Component | ToucanForm', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(<template><ToucanForm data-toucan-form /></template>);
+
+    assert.dom('[data-toucan-form]').exists();
+    assert.dom('[data-toucan-form]').hasTagName('form');
+  });
+
+  test('it allows consumers to render their own components', async function (assert) {
+    await render(<template>
+      <ToucanForm>
+        <div data-custom-content />
+      </ToucanForm>
+    </template>);
+
+    assert.dom('[data-custom-content]').exists();
+  });
+
+  test('it yields a Field from ember-headless-form', async function (assert) {
```

Comment:
I am not opposed to doing this, but I wonder what exactly is the use case you have in mind here? Supporting custom controls that are not provided by `<ToucanForm>`?

Reply:
Yup precisely! I envision folks needing/wanting to build their own custom form controls and by yielding back out `Field` it'll help them to do so.  I could be pre-optimizing though 😉.  What do you think?  If we find this is a real need, we can always add it back later.